firmware/controllers/trigger/trigger_central.cpp

```diff
@@ -509,12 +509,8 @@ extern PwmConfig triggerSignal;
 extern uint32_t hipLastExecutionCount;
 extern uint32_t hwSetTimerDuration;
 
-extern uint32_t maxLockedDuration;
```

Comment:
why do we remove maxLockedDuration? in my opinion this is an important and unrelated feature